tom_nonsidereal_airmass/templatetags/nonsidereal_airmass_extras.py

```diff
@@ -44,3 +48,90 @@ def nonsidereal_target_plan(context):
         'target': context['object'],
         'visibility_graph': visibility_graph
     }
+
+
+@register.inclusion_tag('tom_targets/partials/target_distribution_nonsidereal.html')
+def target_distribution_nonsidereal(targets):
+    """
+    Displays a plot showing on a map the locations of ALL targets in the TOM.
+    Positions are plotted every ten days.
+    """
+    locations = targets.filter(type=Target.SIDEREAL).values_list('ra', 'dec', 'name')
+    data = []
+    for target in targets:
+        if target.type=='NON_SIDEREAL':
```

Comment:
Can you make this `Target.NON_SIDEREAL` the way it is on line 59? We like to avoid magic numbers (or strings, in this case) where we can.

Reply:
Yup!